addon/components/yeti-table/component.js

```diff
@@ -150,13 +150,13 @@ class YetiTable extends DidChangeAttrsComponent {
    * Use this argument to enable the pagination feature. Default is `false`.
    */
   @argument('boolean')
-  pagination = false;
+  pagination = this.get('config').pagination === undefined ? false : this.get('config').pagination;
```

Comment:
I think we can just do `pagination = this.get('config').pagination;` here.

Reply:
wouldn't that possibly leave pagination undefined instead of false, If they didn't have the key in their config? If you always check for true, then yes possibly